client/src/lib/parse/data/PersonAndGroupParser.ts

```diff
@@ -440,6 +442,7 @@ export default class PersonAndGroupParser extends EntityParser {
           type = filteredTypes.length > 0 ? filteredTypes[0] : ''
         }
 
+        console.log(type)
```

Comment:
@kamerynB can you remove the `console.log` here?

Reply:
I actually removed it and I'll merge. Just look out for leftover console.logs from debugging 😉